src/ui/deck.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 The boardgame.io Authors
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import "./deck.css";
+
+const Deck = ({ cards, ...rest }) => (
+  <div className="bgio-deck" {...rest}>
+    {cards.map((card, i) =>
+      React.cloneElement(card, {
+        key: i,
+        className: i > 0 ? "no-hover" : "",
```

Comment:
Let's make `canHover` a prop of `Card` (i.e. don't set class elements on `Card` from `Deck`).